EO-DataHub/eodh-workflows
# Feat/index calculation logic

# Changes
- Adds Index calculation logic
- Adds area size validation
- Allows to run docker container with args

## How to verify
- Create local directory `./data/eodh-results`
- Make sure docker will have rights to save data in this directory: `chmod 777 ./data/eodh-results`
- Run `make docker-build`
- Run `make docker-run`
- The local directory should be populated with data
